community/faas/docs/templates/network.md

```diff
@@ -0,0 +1,40 @@
+# Networks and subnets
+
+Templated network and subnet deployment
+
+## Prerequisites
+- Install [gcloud](https://cloud.google.com/sdk)
+- Create a [GCP project, setup billing, enable requisite APIs](docs/templates/project.md)
+
+
+## Deployment
+
+### Resources
+
+- [compute.v1.network](https://cloud.google.com/compute/docs/reference/latest/networks)
+- [compute.v1.subnetwork](https://cloud.google.com/compute/docs/reference/latest/subnetworks)
+
+
+### Properties
+
+See `properties` section in the schema files
+
+-  [network](../../templates/network.py.schema)
```

Comment:
Is there any way to auto-generate a nice property explanation in the markdown? Like terradoc.

Reply:
That's the idea, but I don't think we're going have time get this done on phase 1. Can we log this task in a ticket and address it later for all templates

Reply:
Yes, please track for later.